packages/backend/vitest.config.js

```diff
@@ -27,9 +27,10 @@ const config = {
     coverage: {
       provider: 'v8',
       reporter: ['lcov', 'text'],
+      extension: '.ts',
```

Comment:
AFAIK it seems this is the fix, so I would rather split into chunks

some are code formatting, this one is a fix and the previous one is a change but usually it's b/c the type is changed  to module

Reply:
Done, commit only fixes vitest.config now.